fre/app/generate_time_averages/generate_time_averages.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
thinking it through- we want users to call this function at the command line, i would hope they use `fre app gen-time-averages`! the current click-based CLI usage is tested for.

we also facilitate using `generate_time_averages` as an importable module- this is tested for!

we have no intention of facilitating direct-execution not-via-click at this time, as i understand it. and we do not test for it's usage, but this is what the she-bang at the top facilitates. so, in order to avoid confusion in use cases, i think we remove it.